src/config/RemoteBrowserTarget.ts

```diff
@@ -37,8 +37,13 @@ export interface CSSBlock {
 
 interface ExecuteParams {
   globalCSS?: string | Array<CSSBlock>;
-  assetsPackage?: unknown;
-  staticPackage?: unknown;
+
+  /** Path to the assets package */
+  assetsPackage?: string;
+
+  /** Path to the static package */
+  staticPackage?: string;
```

Comment:
This is what the worker is expecting, so I decided to keep it named staticPackage here.